Blog: Multi product pricing (#5632)
PostHog/posthog.com: contents/blog/multi-product-pricing.md

The moment we started charging, awareness increased. Users took the feature more seriously, and they shared more demanding feedback – paying users are less bug tolerant!

We put a small team on it, who quickly iterated with customers and improved the product. Session replay took off as a result – it now has similar daily usage to product analytics.

## 2. Focusing on paying customers improves the free product (and drives growth)

We spent the first year and a half of PostHog prioritizing our open-source project – we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product – product analytics.

Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience – a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.

> This journey has since repeated itself with session replay. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements for everyone.
> 
> And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.

Reviewer: ```suggestion
This journey has repeated itself with session replay. 

Since the free and paid session replay products are largely the same, except for a usage limit with the free product, focusing on more demanding, paid users improved the product for everyone and drove word-of-mouth growth.

And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.
```

Reviewer: @charlescook-ph Have tweaked this bit now.

---
date: 2023-04-04
title: What we've learned so far about multi product pricing, and our future plans
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - james-hawkins
featuredImage: ../images/blog/posthog-ceo-diary-blog.png
featuredImageType: full
category: CEO diaries
---

PostHog started as a single product company. We just sold product analytics with a bunch of large extra features thrown in - session replay, feature flagging, experimentation, and even a Customer Data Platform.

We charged by event ingested. Obviously, we wound up with abuse - a handful of high volume users would send zero events, but would use our Javascript snippet to do just session replay, to store _millions_ of recordings.

> We decided a multi product pricing strategy had long term advantages, and - to be frank - we had to stop people abusing recording, and we didn't want to limit it for users already paying us a healthy amount.

Reviewer 2: While less clean, I prefer James' original language here as it feels more authentic (e.g we did have abuse, he wouldn't say 'Q4')

> ## It's a way to take extra products seriously

Reviewer 2: Italics over CAPS?

> ## It's a way to take extra products seriously

Reviewer: Not sure you can / would want to italicise a heading. Probably best to just remove the caps.

The most important thing, but the one we didn't anticipate, was charging for session recording very quickly led to more usage.

Eh? 

Positioning. If you see two products listed side-by-side on a website, you know each is being treated as a first class citizen. Beforehand, we were "product analytics++". That meant no one was taking our session replay feature seriously.

The moment we started charging, more users realized the feature existed, the more seriously they took it, and the more demanding their feedback. Paying users are less bug tolerant!

We put a small team on it, who quickly iterated with customers and improved the quality.

Growth of session replay took off - it now has similar daily usage to product analytics.

> ## Focusing on paying customers can improve the free product

Reviewer: ```suggestion
## 2. Focusing on paying customers benefits everyone (and drives growth)
```

> Intuitively, if someone is paying, they will have a higher expectation.
> 
> However, the secondary effect we didn't anticipate is that fixing the smaller issues can lead to much more usage from both paid and free users. It turns out that in 2021, a lot of UX weirdness and buginess we had was affecting all users of our platform, but our free users weren't complaining that loudly.
> 

Reviewer: ```suggestion
```

> As context, PostHog spent the first year and a half prioritizing our open source project - we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product - product analytics.

Reviewer: ```suggestion
We spent the first year and a half of PostHog prioritizing our open-source project – we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product – product analytics.
```

Reviewer 2: Again, James here was messier but more authentic? ('Leverage baby')

> Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience - a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.

Reviewer: ```suggestion
Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience – a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.
```

> This journey has since repeated itself with session recording. We have a small team of 2 working on this product, and we set them the goal in Q4 last year of charging for their product and getting their first paying customers. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements. 

Reviewer: ```suggestion
This journey has since repeated itself with session replay. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements for everyone.
```

Reviewer 2: I'm assuming you removed the reference to 2 people and charging revenue because it feels scary to put this out there?

Reviewer: No, was more to avoid repetition / contradiction from the previous section where we talk about putting a small team on it + just to shorten the section a bit.

> And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.

Reviewer: I feel like these opening two sections are a little too similar. The second one just rehashes points from the first (i.e. paid users give better feedback) in a slightly different, more expansive way.

Perhaps it's better to reshape the first to focus purely on how / why charging increased usage, while the second digs into how we got better feedback from paying customers, which improve the product for everyone.

@jamesefhawkins I'm happy to do those tweaks given you're pressed for time. It won't take much.

> And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.

Reviewer: ```suggestion
Those quality improvements have driven word-of-mouth growth for our session recording feature alone, adding momentum to our growth engine. And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable. Win-win.
```

Reviewer 2: Your adds sound corporate-y again. (Sorry, broken record but I don't think James would ever say 'adding momentum to our growth engine')

Reviewer: Fair. I'll find a better way to say that.

> 

Reviewer: ```suggestion
## 3. It gives our small teams more autonomy

One of the core learnings we've already had is that a small team structure suits a multi-product approach really well. The concept is a small, multi-disciplinary team should be able to ship into production with as little outside interference as possible. Each team is designed to work like a startup. 

Our reasoning? If we want a wide product, we need to optimize for speed over control... and startups get the most functionality built per person. You won't manage to compete on both _polish_ via controlling a team more tightly _and_ being wider at the same time, which requires autonomy.

Our pricing plans feed into this. In the long run, we want each small team to have control over their pricing – they've got all the user context after all, and they can get some coaching on how to do this from others in the company if needed.
```

> ## Our current thining - outcompeting on price, sustainably
> 
> Do you want to buy 5 individual products at $10 each, or all 5 for $20?
> 
> Charging by product means you can outcompete all of your competitors on price.
> 
> To get to this point, we're starting to build data on cross-selling to know that we'd get it right, so we'll figure that out over the next year and hopefully will be able to lower prices as a result.
> 
> If you're thinking about applying this concept to your business, success depends on where in the market you focus (you need to be able to sell more to the same users, ideally - to avoid a complex sales motion) and how price sensitive they are. If people don't care about price in your market, there's no point either.

Reviewer: ```suggestion
## 4. It allows us to compete on price sustainably

Do you want to buy 5 individual products at $10 each, or all 5 for $20?

Charging by product means you can outcompete all of your competitors on price.

To get to this point, we're starting to build data on cross-selling to know that we'd get it right, so we'll figure that out over the next year and hopefully will be able to lower prices as a result.

If you're thinking about applying this concept to your business, success depends on where in the market you focus (you need to be able to sell more to the same users to avoid a complex sales motion) and how price sensitive they are. If people don't care about price in your market, there's no point either.
```

> ## More autonomy for small teams
> 
> One of the core learnings we've already had is that a small team structure suits a multi product approach really well. The concept is a small, multi-disciplinary team should be able to ship into production with as little outside interference as possible. Each team is designed to work like a startup. Our reasoning? If we want a wide product, we need to optimize for speed over control... and startups get the most functionality built per person. You won't manage to compete on both _polish_ via controlling a team more tightly _and_ being wider at the same time, which requires autonomy.
> 
> Our pricing plans feed into this. In the long run, we want each small team to have control over their pricing - they've got all the user context after all, and they can get some coaching on how to do this from others in the company if needed.

Reviewer: ```suggestion
```

> ## Long term it's a clever way to scale exponentially, we think

Reviewer: ```suggestion
## 5. It's a clever way to scale exponentially (we think)
```

We charged for session recording in December. By March, it was 10% of our revenue. Most of the users paying are new.

That sounds good, but in absolute terms - we'd get double the revenue from it with the same engineering input if we were twice the size when we launched it. As we get bigger and we ship more products, each one will get more traction with the same cost to deliver it. And, in fact, each new product brings us more top-of funnel growth - since it means there are more reasons for people to find PostHog in the first place.

> ## What's the catch

Reviewer: ```suggestion
## What's the catch?
```
